srcpkgs/NetworkManager/template

```diff
@@ -68,6 +67,16 @@ pre_configure() {
 	fi
 }
 
+do_check() {
+    if [ "$XBPS_CHECK_PKGS" = full ]; then
+        : ${make_cmd:=ninja}
+        : ${make_check_target:=test}
+        : ${meson_builddir:=build}
+
+        ${make_cmd} -C ${meson_builddir} ${makejobs} ${make_check_args} ${make_check_target}
```

Comment:
```suggestion
        ninja -C build test
```

Reply:
Yeah, seems better. I just copied it from the buildstyle. Will change that